lmms_eval/tasks/video-tt/utils.py

```
@@ -0,0 +1,279 @@
+import datetime
```

Ruff (0.11.9)

`datetime` imported but unused

Remove unused import: `datetime`

(F401)